packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -304,6 +304,17 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
             </div>
         </div>
 
+        <oui-message
+            type="info"
+            data-ng-if="$ctrl.process.selectedOffer.changeTechno"
+        >
+            <p
+                class="mb-0"
+                data-ng-repeat="line in $ctrl.TEMPORARY_PACK_LINES track by $index"
```

Comment:
```suggestion
                data-ng-repeat="line in [1,2,3] track by $index"
```
If you want reduce the code, in this case i think we can avoid create a constant for just 3 number and directly put the array with 1,2,3